Core/Database/Repositories/EmpireRepository.cs

```diff
@@ -13,16 +14,34 @@ public interface IEmpireRepository
 public class EmpireRepository : IEmpireRepository
 {
     private readonly IDbConnection _db;
+    private readonly ICacheManager _cacheManager;
 
-    public EmpireRepository(IDbConnection db)
+    public EmpireRepository(IDbConnection db, ICacheManager cacheManager)
     {
         _db = db;
+        _cacheManager = cacheManager;
     }
 
     public async Task<byte?> GetEmpireForAccountAsync(Guid accountId)
```

Comment:
I would make this `GetEmpireForPlayerAsync` because technically it should be possible to select an empire per player. The client is blocking this currently but it should be possible imo